services/frontend-service/src/ui/components/tooltip/tooltip.tsx

```diff
@@ -26,7 +26,7 @@ export const Tooltip = (props: { children: JSX.Element; tooltipContent: JSX.Elem
             <a href={href} id={'tooltip' + id} data-tooltip-place="bottom" data-tooltip-delay-hide={delayHide}>
                 {children}
             </a>
-            <TooltipReact className={'tooltip'} anchorSelect={'#tooltip' + id} border={'2px solid lightgray'}>
+            <TooltipReact className={'tooltip'} anchorSelect={'#tooltip' + id}>
```

Comment:
I ran this, and the border is missing now. Please restore it, the border makes the tooltip more readible.

Reply:
Ah, you are right. It also creates a dozen error logs :(